src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/ChargeCommand.java

```diff
@@ -0,0 +1,54 @@
+package io.github.thebusybiscuit.slimefun4.core.commands.subcommands;
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.Rechargeable;
+import io.github.thebusybiscuit.slimefun4.core.commands.SlimefunCommand;
+import io.github.thebusybiscuit.slimefun4.core.commands.SubCommand;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+/**
+ * {@link ChargeCommand} adds an in game command which charges any {@link Rechargeable}
+ * item to max.
```

Comment:
You could be more specific here *"to the maximum value as defined by Rechargeable#getMax..."